Types error: Parameter 'languageType' implicitly has an 'any' type

The most recent release broke TypeScript support:

```
node_modules/languagedetect/index.d.ts(55,25): error TS7006: Parameter 'languageType' implicitly has an 'any' type.
```

Seems like the types file has a small error [here](https://github.com/FGRibreau/node-language-detect/blob/master/index.d.ts#L55). I'll send a PR to fix it.

--- a/index.d.ts
+++ b/index.d.ts
@@ -52,7 +52,7 @@ declare module 'languagedetect' {
          * setLanguageType('iso2');
          * @param languageType
          */
-        setLanguageType(languageType): void;
+        setLanguageType(languageType: string): void;
     }
 
     export = LanguageDetect;

